### DISCUSSION TITLE
Recurring invoices

### DISCUSSION THREAD

**Discussion by dcowan-london** [baimard/gestion#353] [Ideas]
Hi,

Been having a look at this, and just wondering if recurring invoices are planned at all?

Thanks!

**Comment by baimard** [comment]
Taken into account, maybe with a duplicate button

**Comment by VA2XJM** [comment]
A must have for support services. I add my voice to this request.

Idéal pour les services de support. J'ajoute ma voix à cette requête.

**Comment by baimard** [comment]
Merci :)

**Comment by baimard** [comment]
It's plan
